### ISSUE TITLE
Feature Request: Racial bonuses wise science spending rates autodistribution

### ISSUE THREAD

**Issue by 1oom-fork** [1oom-fork/1oom#74]
<sup>Originally posted by **sytnik-aa** March 23, 2024</sup>
I really like the feature when you press the "=" key and all science spending ratios are set to equal. 

<img width="725" alt="image" src="https://github.com/1oom-fork/1oom/assets/40230135/4525a01d-a090-4da5-9c83-fc26e475bbc5">

This feature may be even more interesting if it takes into account racial differences in technology development (page 57 of the manual).

<img width="348" alt="image" src="https://github.com/1oom-fork/1oom/assets/40230135/0160b6c0-fab2-4de3-bf56-11fcba80c97a">

For example, equal spendings are 16,67% per technology field. 

But the adjusted equal spendings for Alkari (since they poor in Force Fields and excellent in Propulsion) are: 
x + x + 1.25x + x + 0.6x + x = 5.85x; 
x = 17.09%

Computers = 17.09%
Construction = 17.09%
Force Fields = 17.09% * 1.25 = 21.37%
Planetology = 17.09%
Propulsion = 17.09% * 0,6 = 10.25%
Weapons = 17.09%

I think it would be great if the first press on "=" key set spendings as equal and the second one -- as adjusted-equal. 

Thanks!

**Comment by 1oom-fork** [comment]
I opened this topic again because I found a paradigm that allows us to consider these innovations as acceptable.
Some features, such as the research bonus highlight and the spy hint during technology theft, are simplifications of the original rules and reduce the demands on players' mental abilities. For me, such consequences are unacceptable, however, cheats have always existed for lazy people, and therefore I thought that unacceptable simplifications could be called cheats and added to the game, labeling those who use them as cheaters. This approach should suit both those who are used to solving the problem themselves, and those who want to relax as much as possible and don’t worry about labels.
I'm not ready to consider your proposed feature right now, but I no longer think it's bad.
